### DISCUSSION TITLE
Making a bot army?

### DISCUSSION THREAD

**Discussion by Kuga-for-real** [PrismarineJS/mineflayer#2695] [Q&A]
Hello I tried making a while loop to spawn a certain amount of bots, each with a random name,
I tried a-lot and I couldn't do it. Does someone have a script for this? 
Thanks.

**Comment by amoraschi** [comment]
Look at `examples/multiple.js`

**Reply by Kuga-for-real** [reply]
**Hello and thanks for your help,**
```
Error: read ECONNRESET
    at TCP.onStreamRead (node:internal/stream_base_commons:217:20) {
  errno: -54,
  code: 'ECONNRESET',
  syscall: 'read'
}
Error: read ECONNRESET
    at TCP.onStreamRead (node:internal/stream_base_commons:217:20) {
  errno: -54,
  code: 'ECONNRESET',
  syscall: 'read'
}
```
This errors show up when running multipe.js (I didn't edit any of the code)
and I ran it using the node multipe.js host port
and it doesn't work? 
Did I do something wrong?
Sorry If i bother you.

**Reply by amoraschi** [reply]
You actually need to supply it with a real server host and port

**Reply by Kuga-for-real** [reply]
> You actually need to supply it with a real server host and port

I know lol,
That is what I did using my server, I just did not use it here for privacy reasons.
